src/utils/request/parseBody.ts

```diff
@@ -11,7 +11,7 @@ export function parseBody(body?: MockedRequest['body'], headers?: Headers) {
     return body
   }
 
-  const contentType = headers?.get('content-type') || ''
+  const contentType = (headers?.get('content-type') || '').toLowerCase()
```

Comment:
That's a nice catch. What do you think if we utilize optional chaining here since we're already using it?

```suggestion
  const contentType = headers?.get('content-type')?.toLowerCase() || ''
```

Reply:
Sure 👍🏽